dashboard/app/controllers/transfers_controller.rb

```diff
@@ -4,71 +4,51 @@ class TransfersController < ApplicationController
 
   # POST /sections/:id/transfers
   def create
-    # TODO(asher): Much of the permissioning here should be done through CanCan.
     new_section_code = params[:new_section_code]
+    current_section_code = params[:current_section_code]
+    unless new_section_code && current_section_code
+      return head :bad_request
+    end
 
-    new_section = Section.find_by_code(new_section_code)
-    unless new_section
-      render json: {
-        error: I18n.t('move_students.new_section_dne', new_section_code: new_section_code)
-      }, status: :not_found
-      return
+    stay_enrolled_in_current_section = params[:stay_enrolled_in_current_section].try(:to_bool)
+    if stay_enrolled_in_current_section.nil?
+      return head :bad_request
```

Comment:
Yes (as I note in the PR summary), this change is intentional. Its only [usage](https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml#L510) includes the parameter.